R/data.table.R

```diff
@@ -2674,6 +2674,7 @@ chgroup = function(x) {
 }
 
 rbindlist = function(l, use.names="check", fill=FALSE, idcol=NULL) {
+  if(!is.list(l)) stop("Input is not a list")
```

Comment:
add extra line above
```r
if (is.null(l)) return(null.data.table())
```
this will fix failure of test 682

also please use extra whitespace between `if` and `(`, so `if (...) ...`

Reply:
hi, i've made those two changes.  thanks for making this a cinch!